examples/movie_view_ratings/run_all_frameworks.py

```diff
@@ -79,12 +80,22 @@ def calc_dp_rating_metrics(movie_views, backend, public_partitions):
         min_value=1,
         max_value=5)
 
+    value_extractor = lambda mv: mv.rating
+
+    if FLAGS.vector_metrics:
```

Comment:
@dvadym , I've added a flag to compute vector metrics over the ratings values, which are extracted as vectors with zeros and one in the position of the rating. What other example would be useful to add?